### DISCUSSION TITLE
Which wallet to use?

### DISCUSSION THREAD

**Discussion by ghost** [catchthatrabbit/coreminer#77] [Wishlist]
Which wallet for core miner?

**Comment by StotheC90** [comment]
U need to use the ICAN wallet generator, to create a new Wallet for mining ;)

**Reply by rastislavcore** [reply]
[Wallet generator][1] is the best for now till [CorePass][2] will be live.

[1]: https://github.com/core-coin/wallet-generator
[2]: https://corepass.net/
